docs/buildah-push.md

```diff
@@ -45,6 +45,10 @@ If the transport part of DESTINATION is omitted, "docker://" is assumed.
 
 ## OPTIONS
 
+**--all-tags, a**
```

Comment:
Should be a `-a`

Reply:
fixed, will touch up in pull too.  Thx for the eagle eye.